### PR TITLE
Moving CommentViewHolder binding logic away from the Activity

### PR DESCRIPTION
**Pull request by florina-muntenescu**

### CODE REVIEW

File: designernews/src/main/java/io/plaidapp/ui/designernews/story/CommentViewHolder.kt [hunk 1]
```diff
@@ -130,4 +149,18 @@ internal class CommentViewHolder(itemView: View) : RecyclerView.ViewHolder(itemV
...
         }
     }
+
+    fun setExpanded(expanded: Boolean) {
+        if (expanded) {
+            val threadDepthWidth = threadDepth.getDrawable().getIntrinsicWidth()
+            val leftShift = (-(threadDepthWidth + (threadDepth.getLayoutParams() as ViewGroup.MarginLayoutParams).marginEnd)).toFloat()
+            author.translationX = leftShift
+            comment.translationX = leftShift
+            threadDepth.translationX = (-(threadDepthWidth + (threadDepth.getLayoutParams() as ViewGroup.MarginLayoutParams).marginStart)).toFloat()
```
Comment (keyboardsurfer) [inline designernews/src/main/java/io/plaidapp/ui/designernews/story/CommentViewHolder.kt]: Can be casted once by holding on to a `val`.
Would make this calculation easier to read.

File: designernews/src/main/java/io/plaidapp/ui/designernews/story/CommentViewHolder.kt [hunk 2]
```diff
@@ -28,13 +28,32 @@ import io.plaidapp.core.util.AnimUtils.getFastOutSlowInInterpolator
...
+    private val threadGap: Int
+) : RecyclerView.ViewHolder(itemView) {
+
+    private val threadDepth: ImageView = itemView.findViewById(R.id.depth)
+    private val author: AuthorTextView = itemView.findViewById(R.id.comment_author)
+    private val timeAgo: TextView = itemView.findViewById(R.id.comment_time_ago)
+    val comment: TextView = itemView.findViewById(R.id.comment_text)
 
-internal class CommentViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+    init {
```
Comment (keyboardsurfer) [inline designernews/src/main/java/io/plaidapp/ui/designernews/story/CommentViewHolder.kt]: Why not use `apply` during initialization above?

File: designernews/src/main/java/io/plaidapp/ui/designernews/story/DesignerNewsStory.java [hunk 3]
```diff
@@ -527,6 +521,25 @@ private void bindDescription() {
         }
     }
 
+    private CharSequence getStoryPosterTimeText(
```
Comment (keyboardsurfer) [inline designernews/src/main/java/io/plaidapp/ui/designernews/story/DesignerNewsStory.java]: We're still in Java country here.

File: designernews/src/main/java/io/plaidapp/ui/designernews/story/CommentViewHolder.kt (hunk 2 above)
Comment (florina-muntenescu) [reply designernews/src/main/java/io/plaidapp/ui/designernews/story/CommentViewHolder.kt]: doesn't work

**Pull request merged**